content/tutorials/earthworks/basics.qmd

```diff
@@ -0,0 +1,403 @@
+---
+title: "Basic earthworks"
```

Comment:
Perhaps standardize the titles so it's clear they are related, e.g. Terrain modeling with r.earthworks: Basics

Reply:
I think Terrain modeling is rather general, so I would prefer r.earthworks: ... since it's very specific to it.

I can't tell what would be best for the noise tutorial, but it can be always linked from the r.earthworks tutorials but be standalone.